aries-site/src/pages/templates/inline-notifications.mdx

```diff
@@ -0,0 +1,143 @@
+import { Example } from '../../layouts';
+import { InlineNotificationExample } from '../../examples';
+import { Checkmark } from 'grommet-icons';
+
+Inline notifications are messages that appear near the content to which they
+are related. They may be task or system generated, communicating information
+such as status updates, alerts to success or failure of an action, or call
+attention to a new feature in which a user may be interested.
+
+<Example
+  code=""
+  github=""
+  showResponsiveControls={['fullScreen']}
+  screenContainer
+  height="650px"
+  width="100%"
+>
+  <InlineNotificationExample />
+</Example>
+
+## When to use
+
+Consider using an inline notification when presenting:
+
+- **Feedback** &ndash; Providing feedback to users about their actions. For
+  example, if a user submits a form requiring server-side validation, an
+  inline notification should be used to display and summarize form-level errors.
+- **Significant status updates** &ndash; Delivering significant status updates
+  about a task. For example, if a user initiated an ansynchronous process such
+  as a device update, a notification might provide the status update upon.
+  "Success - all devices have updated to the latest version."
+- **Promotions** &ndash; Calling a user's attention to such as new features, tips,
```

Comment:
Should we flag that this should be used sparingly? Or responsibly? Something to indicate it shouldn't be reached for all the time (I think people know that but just in case)

Reply:
Done.